components/ws-proxy/pkg/proxy/infoprovider.go

```diff
@@ -6,65 +6,59 @@ package proxy
 
 import (
 	"context"
-	"io"
 	"net/url"
-	"strconv"
-	"sync"
+	"strings"
 	"time"
 
-	validation "github.com/go-ozzo/ozzo-validation"
 	"golang.org/x/xerrors"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/credentials"
-
-	common_grpc "github.com/gitpod-io/gitpod/common-go/grpc"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/cache"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"github.com/gitpod-io/gitpod/common-go/kubernetes"
 	"github.com/gitpod-io/gitpod/common-go/log"
-	"github.com/gitpod-io/gitpod/common-go/util"
+	regapi "github.com/gitpod-io/gitpod/registry-facade/api"
+	"github.com/gitpod-io/gitpod/ws-manager/api"
 	wsapi "github.com/gitpod-io/gitpod/ws-manager/api"
 )
 
-// WorkspaceCoords represents the coordinates of a workspace (port)
+const (
```

Comment:
this feels like a duplicate. Don't we have those somewhere in `common-go` already? :thinking:

Reply:
@geropl yes, this const are from `ws-manager` but all const are private. Not sure what's the right thing to do here.

Reply:
moving them to common-go would be in line with how we've done this with other annotations

Reply:
done